### DISCUSSION TITLE
How to login new master data us folder in application so please give All steps. so help community team.

### DISCUSSION THREAD

**Discussion by Amitsinghongraph** [egovernments/Digit-Core#273] [DIGIT Platform]
How to login new master data us folder in application so please give All steps. so help community team.

**Comment by shashwat-egov** [comment]
Hi Amit, please refer to this documentation for knowing more about how MDMS service works and how to add new master data - [MDMS documentation](https://core.digit.org/platform/core-services/mdms-master-data-management-service)
